### DISCUSSION TITLE
new logical condition: 'for all but'

### DISCUSSION THREAD

**Discussion by Billy60** [FrSkyRC/ETHOS-Feedback-Community#4699] [Q&A]
Bertrand, in Otx there is a conditional feature which was very useful: you could get an action for "all cases but". That means you get a 'yes' if only one condition is fulfilled and none of the others. Useful for instances in switches. Are you planning to implement this in Ethos ?

Thanks a lot and congratulations about your outstanding work !

**Comment by bsongis** [comment]
Perhaps you are missing the hamburger icon in the whole Ethos?

![image](https://github.com/user-attachments/assets/cdadc7ef-cdce-43e4-903f-05a91765ef0b)

**Comment by Billy60** [comment]
Yes ! thank you, I had not connected !
Please note I am purchasing a Tandem XE because of the quality of your Ethos. Makes the difference !

**Comment by bsongis** [comment]
Say it to my boss ;)
